src/ensembl/src/content/app/custom-download/components/checkbox-with-radios/CheckboxWithRadios.test.tsx

```diff
@@ -34,47 +36,60 @@ describe('<CheckboxWithRadios />', () => {
     jest.resetAllMocks();
   });
 
-  let wrapper: any;
   const defaultProps = {
     onChange: onChange,
     label: 'foo',
     selectedOption: '',
     options: times(5, () => createOption())
   };
 
+  const renderCheckboxWithRadios = (props?: Partial<CheckboxWithRadiosProps>) =>
+    render(<CheckboxWithRadios {...defaultProps} {...props} />);
+
   it('renders without error', () => {
-    wrapper = mount(<CheckboxWithRadios {...defaultProps} />);
-    expect(wrapper.find(CheckboxWithRadios).length).toEqual(1);
+    expect(() => {
+      renderCheckboxWithRadios();
+    }).not.toThrow();
   });
 
   it('does not check the checkbox when there are no options selected', () => {
-    wrapper = mount(<CheckboxWithRadios {...defaultProps} />);
+    const { container } = renderCheckboxWithRadios();
 
-    expect(wrapper.find(Checkbox).prop('checked')).toBe(false);
+    expect(container.querySelector('.checkboxHolder .checked')).toBeFalsy();
   });
 
   it('does not display any radios when the checkbox is unchecked', () => {
-    wrapper = mount(<CheckboxWithRadios {...defaultProps} />);
+    const { container } = renderCheckboxWithRadios();
+
+    expect(container.querySelector('.checkboxHolder .checked')).toBeFalsy();
 
-    expect(wrapper.find(Checkbox).prop('checked')).toBe(false);
-    expect(wrapper.find('input[type="radio"]').length).toBe(0);
+    expect(container.querySelectorAll('input[type="radio"]').length).toBe(0);
```

Comment:
Why not do it the same way as on line 64?

Reply:
That's done